static/js/brand-store/Settings/Settings.test.tsx

```diff
@@ -1,31 +1,19 @@
 import React from "react";
+import * as reactRedux from "react-redux";
 import { BrowserRouter as Router } from "react-router-dom";
 import { Provider, useSelector } from "react-redux";
 import { render, screen } from "@testing-library/react";
 import "@testing-library/jest-dom";
 import Settings from "./Settings";
 import store from "../store";
 
-jest.mock("react-redux", () => ({
-  ...jest.requireActual("react-redux"),
-  useSelector: jest.fn(),
-}));
-
 jest.mock("react-router-dom", () => ({
   ...jest.requireActual("react-router-dom"),
   useParams: jest.fn().mockReturnValue({ id: "test" }),
 }));
 
-let initialState = {};
-
-const setupMockSelector = (state) => {
-  useSelector.mockImplementation((callback) => {
-    return callback(state);
-  });
-};
-
-beforeEach(() => {
-  initialState = {
+function getInitialState() {
```

Comment:
Can you type the return of this function...
```suggestion
function getInitialState(): State {
```